lighthouse-treemap/app/src/util.js

```diff
@@ -14,14 +14,14 @@
 class TreemapUtil {
   /**
    * @param {LH.Treemap.Node} node
-   * @param {(node: LH.Treemap.Node, path: string[]) => void} fn
+   * @param {(node: NodeWithElement, path: string[]) => void} fn
    * @param {string[]=} path
    */
   static walk(node, fn, path) {
     if (!path) path = [];
     path.push(node.name);
 
-    fn(node, path);
+    fn(/** @type {NodeWithElement} */ (node), path);
```

Comment:
isn't this not necessary if L16 is `@param {NodeWithElement} node` ?

Reply:
oh right, because optional.

Reply:
oh, I was also assuming that was coming from the webtreemap library so is already augmented?